src/guiFormSpecMenu.h

```diff
@@ -143,6 +143,16 @@ class GUIFormSpecMenu : public GUIModalMenu
 		{
 		}
 		ImageDrawSpec(const std::string &a_name,
+				const std::string &a_item_name,
+				v2s32 a_pos, v2s32 a_geom):
+			name(a_name),
+			item_name (a_item_name),
+			pos(a_pos),
+			geom(a_geom)
+		{
+			scale = true;
+		}
+		ImageDrawSpec(const std::string &a_name,
 				v2s32 a_pos, v2s32 a_geom):
```

Comment:
please pass v2s32 as const v2s32& to improve a little bit the performance. We don't need to copy those memory places here